ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -365,6 +373,7 @@ export default class TestDataModal extends React.Component {
       },
```

Comment:
I was actually referring to adding an entry here, in this existing `setState()`.